lang/en/tool_mergeusers.php

```diff
@@ -110,3 +110,9 @@
 $string['userselecttable_legend'] = '<b>Select users to merge</b>';
 $string['viewlog'] = 'See merging logs';
 $string['wronglogid'] = 'The log you are asking for does not exist.';
+$string['lastmergetomefrom'] = 'User last merged into this account';
+$string['lastmergefrommeto'] = 'User this account was last merged into';
+$string['nolastmerge'] = 'None';
+$string['unknownprofile'] = 'Unknown userid {$a}';
+$string['usermergedat'] = '{$a->profilelink} on {$a->time}. {$a->loglink}';
```

Comment:
I think you missed a letter: `usermergedata`, or even better as `usermergedetail`.